docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -279,212 +216,237 @@ server, DHCP server, RADIUS server) using:
+ * Step 7 - Configuration - start all services.
 
-[options="compact"]
-* Everyone behind an inline interface is on the same Layer 2 LAN
-* Every packet of authorized users goes through the PacketFence server increasing the server's load considerably: Plan ahead for capacity
-* Every packet of authorized users goes through the PacketFence server: it is a single point of failure for Internet access
-* Ipset can store up to 65536 entries, so it is not possible to have a inline network class upper than B
+Once all services are started, you will automatically be redirected to the PacketFence's web admin interface. It is located at https://@ip_of_packetfence:1443/admin
 
-This is why it is considered a poor man's way of doing access control. We have avoided it for a long time because of the above mentioned limitations. That said, being able to perform both inline and VLAN enforcement on the same server at the same time is a real advantage: it allows users to maintain maximum security while they deploy new and more capable network hardware providing a clean migration path to VLAN enforcement.
+Open that link and log in using the username/password specified in Step 5.
 
-Technical introduction to Out-of-band enforcement 
--------------------------------------------------
+Connecting PacketFence to Microsoft Active Directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Introduction
-~~~~~~~~~~~~
+Next we join the PacketFence server to your existing Microsoft Active Directory domain controller. From PacketFence's web admin interface, go in 'Configuration->Policies and Access Control->Domains->Active Directory Domain' and click on the 'Add domain' button. Provide the required fields. You will need an Active Directory administrative username and password to join the PacketFence server to your domain. Once all the information has been provided, click on the 'Save and join" button'.
 
-VLAN assignment is currently performed using several different techniques. These techniques are compatible one to another but not on the same switch port. This means that you can use the more secure and modern techniques for your latest switches and another technique on the old switches that doesn't support latest techniques. As it's name implies, VLAN assignment means that PacketFence is the server that assigns the VLAN to a device. This VLAN can be one of your VLANs or it can be a special VLAN where PacketFence presents the captive portal for authentication or remediation.
+Once the domain join succeeds, click on the 'Realms' tab. Click on the 'Default' realm and set the domain to the Active Directory domain you have just created. That will instruct PacketFence to use that newly created Active Directory for the default authentication realm.
```

Comment:
I'd also add it to the NULL realm by saying at the end:
"Next, do the the for the NULL realm."

Reply:
do the the the the the the?

Reply:
More like

do the the datatababazzeee